mixitup/index.html

```diff
@@ -807,6 +109,8 @@ <h3 class="nameOnModal">Star Ying</h3>
         <script src='js/jquery.js'></script>
         <script src='js/jquery.mixitup.js'></script>
         <script src='js/bootstrap.js'></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.min.js"></script>
```

Comment:
Since we are hosting all the other dependencies, should we do the same for `handlebars` instead of using a CDN?